File: bruteForce/phoneNumber.cpp
#include <iostream>
#include <fstream>
#include <stdio.h>
#include <string>
using namespace std;
int main()
{
    string starts[] = {
        "050",
        "052",
        "053",
        "054",
        "058",
        "059",
        "072",
        "073"};
    ofstream file("PhoneNumbers.txt");
    for (string start : starts)
    {
        for (int i = 0; i < 10000000; i++)
        {
            file << start;
            file << i;
            file << "\n";
        }
        printf("%s\n", start.c_str());
    }
}
string padZero(int num)
{
    string str = to_string(num);
    while (str.length() < 7)
    {
        str = "0" + str;
    }
    return str;
}

File: bruteForce/phoneNumber.rs
use std::fs;

fn main() {
    let phone_prefixes: [&str; 6] = ["050", "052", "053", "054", "058", "059"];

    let mut numbers = String::new();
    for prefix in phone_prefixes {
        for i in 0..10000000 {
            numbers += prefix;
            numbers += &pad_zero(i);
            numbers += "\n";
        }
    }

    let _ = fs::write("PhoneNumbers.txt", numbers);
}

fn pad_zero(num: u32) -> String {
    let mut clone = num.to_string();
    while clone.len() < 7 {
        clone += "0";
    }
    clone
}
